src/views/ForgotPasswordPageChange/index.js

```diff
@@ -5,113 +5,142 @@ import Button from "react-bootstrap/Button";
-          <Form.Control
-            required
-            name="password"
-            type="password"
-            placeholder=""
-            onChange={this.handleChange}
-          />
-        </Form.Group>
-        <Form.Group controlId="inputForm.confirmPassword">
-          <Form.Label>Confirm Password</Form.Label>
-          <Form.Control
-            required
-            name="confirmpassword"
-            type="password"
-            placeholder=""
-            onChange={this.handleChange}
-          />
-        </Form.Group>
-        <Button
-          className="move=btn"
-          variant="success"
-          type="submit"
-          onClick={this.handleSubmit.bind(this)}
-        >
-          <Link className="btn-link" to="/forgotpassword">
-            Change Password
-          </Link>
-        </Button>
-        <div className="display-error" id="display_error"></div>
-        <Route path="/LoginPage">
-          <LoginPage />
-        </Route>
-      </Form>
+      <div>
+        <div style={{ backgroundColor: "#EEEEEE", margin: "auto", borderRadius: "16px", overflow: "hidden", boxShadow: " 0 19px 38px rgba(0, 0, 0, 0.30), 0 5px 5px rgba(0, 0, 0, 0.22)" }}>
+          <div style={{ backgroundColor: "#EEEEEE", margin: "0px 200px 0px 200px", display: "flex", flexDirection: "column", justifyContent: "center", alignItems: "center", padding: "100px 0px 100px 0px" }}>
+            <style type="text/css">
+              {`
+            .form-control {
+              background-color: #C4C4C4;
+              color: black;
+              border-radius: 16px;
+            }
+            .form-control:focus{
+              background-color: #C4C4C4;
+              color: black;
+            }
+            body {
+              margin: 0;
+              display: flex;
+              flex-direction: column;
+              background-color: #C5CFEA;
+              height: 100%;
+              justify-content: center;
+              align-items: center;
+              background-image: linear-gradient(#087a99, #053E5E);
+              display: flex;
+              justify-content: center;
+              align-items: center;
+              height: 100%;
+            }
+          html { height: 100%; }
+          svg {
+            max-height: 100%;
+          }
+          .submit-btn{
+            border-radius: 16px;
+            background-color: #18CF52;
+          }
+          .info {
+            justify-content: center;
+            align-items: center;
+            flex-direction: row;
+            white-space: nowrap;
+            position: relative;
+            text-align: center;
+          }
+          .logo {
+            padding: 0px 0px 0px 0px;
+            float: left;
+            position: absolute;
+            left: -120px;
+          }
+          form{
+            display:flex;
+            flex-direction: column;
+            justify-content: flex-start;
+            min-width: 500px;
```

Comment:
Always try to avoid absolute positioning (ie using px for widths, margins, and padding) since it won't be consistent across different screen sizes and resolutions. Instead, look into using [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#flexbox-background)